crates/re_types/definitions/rerun/blueprint/components/lock_range_during_zoom.fbs

```diff
@@ -13,11 +13,10 @@ namespace rerun.blueprint.components;
 ///
 /// Default is `false`, i.e. zoom will change the visualized range.
 struct LockRangeDuringZoom (
-    "attr.arrow.transparent",
     "attr.rerun.scope": "blueprint",
-    "attr.rust.derive": "Copy, PartialEq, Eq",
+    "attr.rust.derive": "Copy, Default, PartialEq, Eq, PartialOrd, Ord",
     "attr.rust.repr": "transparent",
     "attr.rust.tuple_struct"
 ) {
-    lock_range: bool (order: 100);
+    lock_range: rerun.datatypes.Bool (order: 100);
```

Comment:
Why would we sometimes prefer `bool` over `rerun.datatypes.Bool` and the other way around?

Reply:
Should always prefer it in Components. We want to get to a place where components always forward to a datatype. Not only will this get us closer to a place where components are a thin semantic layer, it also means I don't have to implement bool serialization again and again in Python (and it won't be generated either for C++, Rust is the odd one out where we regenerate a lot of stuff all the time, to be fixed).
We're not consequent in this yet, but it's already happening e.g. with `PanelExpanded`